website/content/plugins/aws-lambda.mdx

```diff
@@ -0,0 +1,94 @@
+---
+layout: plugins
+page_title: 'Plugin: AWS Lambda'
+sidebar_title: 'aws-lambda'
+description: 'Build, Release, and Deploy on AWS Lambda'
+---
+
+# AWS Lambda
+
+## Image Support Only
+
+The AWS Lambda deploy plugin only supports AWS Lambda deployments that utilize ECR images
+rather than .zip files.
+
+We recommend that the `docker` build plugin is used to generate these images.
+
+Below is a simple Dockerfile, handler code, and waypoint.hcl for build, deploying, and
+releasing using Waypoint.
```

Comment:
Make these the same tense "build, deploy, and release" or "building, deploying, and releasing"